project/Dependencies.scala

```diff
@@ -1,12 +1,12 @@
-import sbt._
-import Keys._
+//import sbt._
+//import Keys._
 
-object Version {
-  val jol = "0.5"
-}
-
-object Dependencies {
-  val jol    = "org.openjdk.jol"        % "jol-core" % Version.jol
-  val jolCli = "org.openjdk.jol"        % "jol-cli"  % Version.jol
-}
+//object Version {
+//  val jol = "0.5"
+//}
+//
+//object Dependencies {
+//  val jol    = "org.openjdk.jol"        % "jol-core" % Version.jol
+//  val jolCli = "org.openjdk.jol"        % "jol-cli"  % Version.jol
+//}
 
```

Comment:
Why comment it? This is a style I strongly prefer. Please just bump the version here